src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Square extends Figure {

```

Comment:
The Square class extends Figure but doesn't seem to add any specific properties or methods that differ from the Figure class. Consider adding properties like sideLength and implementing the getArea() method specifically for a square.